packages/account/src/Components/terms-of-use/terms-of-use.scss

```diff
@@ -2,6 +2,8 @@
     display: flex;
     flex-direction: column;
     flex-grow: 1;
+    margin: 0 8rem !important;
+    width: 84% !important;
```

Comment:
will this work in mobile as well?

Reply:
Oops, missed the rest of my changes, sorry about that! Pushed!